source/zh-cn/docs/configuration.md

```diff
@@ -98,6 +98,93 @@ Hexo 使用 [Moment.js](http://momentjs.com/) 来解析和显示时间。
 参数 | 描述
 --- | ---
 `theme` | 当前主题名称。值为`false`时禁用主题
+`theme_config` | 主题的配置文件。在这里放置的配置会覆盖主题目录下的 _config.yml 中的配置
 `deploy` | 部署部分的设置
 `meta_generator` | [Meta generator](https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/meta#%E5%B1%9E%E6%80%A7) 标签。 值为 `false` 时 Hexo 不会在头部插入该标签
 
+### 包括或不包括目录和文件
+
+在 Hexo 配置文件中，通过设置 include/exclude 可以让 Hexo 进行处理或忽略某些目录和文件夹。你可以使用 [glob 表达式](https://github.com/isaacs/minimatch) 对目录和文件进行匹配。
+
+参数 | 描述
+--- | ---
+`include` | Hexo 默认会忽略隐藏文件和文件夹（包括名称以下划线和 `.` 开头的文件和文件夹，Hexo 的 `_posts` 和 `_data` 等目录除外）。通过设置此字段将使 Hexo 处理他们并将它们复制到 `source` 目录下。
+`exclude` | Hexo 会忽略这些文件和目录
+
+举例：
+
+```yaml
+# Include/Exclude Files/Folders
+include:
+  - ".nojekyll"
+  # 包括 'source/css/_typing.css'
+  - "css/_typing.css"
+  # 包括 'source/_css/' 中的任何文件，但不包括子目录及其其中的文件。
+  - "_css/*"
+  # 包含 'source/_css/' 中的任何文件和子目录下的任何文件
+  - "_css/**/*"
+
+exclude:
+  # 不包括 'source/js/test.js'
+  - "js/test.js"
+  # 不包括 'source/js/' 中的文件、但包括子目录下的所有目录和文件
+  - "js/*"
+  # 不包括 'source/js/' 中的文件和子目录下的任何文件
+  - "js/**/*"
+  # 不包括 'source/js/' 目录下的所有文件名以 'test' 开头的文件，但包括其它文件和子目录下的单文件
+  - "js/test*"
+  # 不包括 'source/js/' 及其子目录中任何以 'test' 开头的文件
+  - "js/**/test*"
+  # 不要用 exclude 来忽略 markdown 文件。你应该使用 'skip_render'，或者在 markdown 文件的文件名之前加一个 '_'
+  # 如果你在这里配置一个 - "_posts/hello-world.md" 是没有用的。
+```
+
```

Comment:
please include translation of "Each value in the list must be enclosed with single/double quotes." as well.

Reply:
The missing translation were added in commit https://github.com/hexojs/site/commit/04ff7bb7c12f48eb3b6e4f2a9739d015002328fe